core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -184,6 +188,50 @@ public static <T> void invoke(Logger logger, CompletableFuture<T> future, String
     }
   }
 
+  /**
+   * Calls the given method and handled exception. Exceptions are
```

Comment:
s/Calls the given method and handled exception/
   Calls the given method and handles exception

Reply:
Can you please explain the differences between the `invoke` and `call`?
It seems this is an ASYNC call, right?